releasenotes/notes/logs-send-api-key-in-header-ee99f702687fc6bb.yaml

```
@@ -0,0 +1,4 @@
+---
+security:
```

let's list this under `enhancements`:

```suggestion
enhancements:
```